### ISSUE TITLE
Bug - runtime error in webpack 2.2.0-rc.4 and later

### ISSUE THREAD

**Issue by echenley** [brigand/babel-plugin-flow-react-proptypes#65]
See https://github.com/babel/babel/issues/5155

Basically:

```js
// works
type Props = {}
export type { Props }

// breaks
export type Props = {}
```

I believe this is the Webpack change which breaks this behavior: https://github.com/webpack/webpack/pull/3755. `module.exports` is now undefined when using ES6 modules.

This output is therefore invalid: https://github.com/brigand/babel-plugin-flow-react-proptypes/blob/master/src/__tests__/__snapshots__/export-object-test.js.snap

**Comment by brigand** [comment]
@echenley we could skip exporting the types when exports is undefined. Is that sufficient? I don't want to generate es6 modules, except maybe behind a config flag.

**Comment by echenley** [comment]
Yeah, that works as long as users don't expect to be able to import proptypes when using this plugin? It seems very unlikely that anyone is relying on this:

```js
import { babelPluginFlowReactPropTypes_proptype_Props } from './SomeComponent'
```

**Comment by brigand** [comment]
Yeah, the incredibly long name and lack of documentation discourages use :-)

Published a fix as 0.20.0. Let me know if you have any more issues.

**Comment by echenley** [comment]
Perfect, thank you!
